README.md

```diff
@@ -11,12 +11,15 @@ Essence
 * [Downloads] | [![Github Releases](https://img.shields.io/github/downloads/EssencePowered/Essence/total.svg?style=flat-square)](http://www.github.com/EssencePowered/Essence/releases)
 
 Essence is a Sponge plugin that allows you to quickly start your server with essential commands, events, and other
-tidbits that you might need. Extremely configurable, only loading up the modules you want (and providing a way for
+tidbits that you might need. Extremely configurable, only loading up the commands and modules you want (and providing a way for
 plugins to disable modules that they replace the functionality of), and providing a simple and rich API, Essence is an
 elite plugin for providing simple server tasks.
  
-...or so I hope, soon! With a website full of documentation and everything (at http://quickstart.dualspiral.co.uk/)!
-I'm only just getting this off the ground as Sponge has started to stabilise!
+Essence is the unbeatable combination of HassanS6000's EssentialCmds and experience of writing an essentials plugin, with the 
+system loading technology of dualspiral's QuickStart, making Essence an essential addition to your server.
+ 
```

Comment:
Well, the point is that it _is_ built on my technology for the command and module loader and it _is_ using EssentialCmds' developer. I think it's important that the history of the project is somewhat recognised.

It's all about the marketing.